src/views/DigitalCertificates/FormFields/FormFieldsEditDigitalCertificates.vue

```diff
@@ -35,165 +41,205 @@
-          :label="'Copy'"
-          @click="copyCSRToclipboard"
-        />
-        <small v-if="csrCopied">Copied successfully!</small>
-      </div>
-    </template>
-  </FormHorizontal>
-  <FormHorizontal
-    v-if="!csr && certificateType === certificateTypes.EDGE_CERTIFICATE"
-    title="Update a Server Certificate"
-    description="Paste the PEM-encoded TLS X.509 certificate and private key in the respective fields to update the certificate. The current certificate and private key are hidden to protect sensitive information."
-  >
-    <template #inputs>
-      <div class="flex flex-col sm:max-w-lg w-full gap-2">
-        <label>Name *</label>
-        <InputText
-          v-model="name"
-          type="text"
-          placeholder="My digital certificate"
-          :class="{ 'p-invalid': nameError }"
-        />
-        <small
-          v-if="nameError"
-          class="p-error text-xs font-normal leading-tight"
-          >{{ nameError }}</small
-        >
-      </div>
-      <div class="flex flex-col sm:max-w-lg w-full gap-2">
-        <label>Certificate </label>
-        <PrimeTextarea
-          v-model="certificate"
-          :class="{ 'p-invalid': certificateError }"
-          placeholder="For security purposes, the current certificate isn't exhibited, but it was correctly registered. Paste a new certificate in this field to update it."
-          rows="5"
-          cols="30"
-        />
-        <small
-          v-if="certificateError"
-          class="p-error text-xs font-normal leading-tight"
-          >{{ certificateError }}</small
-        >
-      </div>
-      <div class="flex flex-col sm:max-w-lg w-full gap-2">
-        <label>Private Key </label>
-        <PrimeTextarea
-          v-model="privateKey"
-          :class="{ 'p-invalid': privateKeyError }"
-          placeholder="For security purposes, the current private key isn't exhibited, but it was correctly registered. Paste a new private key in this field to update it."
-          rows="5"
-          cols="30"
-        />
-        <small
-          v-if="privateKeyError"
-          class="p-error text-xs font-normal leading-tight"
-          >{{ privateKeyError }}</small
-        >
-      </div>
-    </template>
-  </FormHorizontal>
+  <template v-if="managed">
+    <FormHorizontal title="Let's encrypt certificate">
+      <template #description>
+        <p>This is a Let's Encrypt™ certificate automatically created and managed by Azion.</p>
+
+        <p>
+          Azion's Certificate Manager is currently verifying if the Domain is correctly pointed by
+          CNAME in the DNS.
+        </p>
+
+        <Divider />
+
+        <div class="flex flex-wrap items-center">
+          <p>
+            If you have not yet pointed a DNS zone, please check the
+            <PrimeButton
+              link
+              class="w-fit p-0 text-sm"
+              icon-pos="right"
+              icon="pi pi-external-link"
+              label="Documentation"
```

Comment:
**nitpick**:
"If you have...Documentation"
Entendo que é um link, mas está com "D" maiúsculo em um texto corrido. E não é um nome próprio. É isso mesmo??

Reply:
peguei o texto da tela do RTM que foi usada como base, lá também foi escrito com o 'D' , vou verificar com o pessoal de UXW , boa.

Vou deixar abaixo o print da tela como consulta futura:

![image](https://github.com/aziontech/azion-console-kit/assets/101186721/46f525be-8ac2-4d9f-ba4f-28c19c590202)